docs/source/examples/recipe_deepdive.rst

```diff
@@ -4,11 +4,11 @@
 Training Recipe Deep-Dive
 =========================
 
-This tutorial will walk you through the design of training-recipes in torchtune.
+This deep-dive will walk you through the design of training-recipes in torchtune.
 
 .. grid:: 1
 
-    .. grid-item-card:: :octicon:`mortar-board;1em;` What you will learn
```

Comment:
(Commenting here cause it's further down in the doc) L190 still needs a change from "config tutorial" -> "config deep-dive"